sdk/batch/Microsoft.Azure.Batch/src/Common/BatchErrorCodeStrings.cs

```diff
@@ -599,56 +599,129 @@ public static class BatchErrorCodeStrings
+        /// <summary>
+        /// Pool creation failed with disk encryption error
+        /// </summary>
         public const string DiskEncryptionFailure = "DiskEncryptionFailure";
-
+        
+        /// <summary>
+        /// A dependent resource specified for the Pool was not found.
+        /// </summary>
         public const string DependentResourceNotFound = "DependentResourceNotFound";
-
+        
+        /// <summary>
+        /// Assigned policy on resource has blocked pool operation.
+        /// </summary>
         public const string OperationsRestrictedByPolicy = "OperationsRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on PublicIP has blocked pool operation.
+        /// </summary>
         public const string OperationsOnPublicIPRestrictedByPolicy = "OperationsOnPublicIPRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on Custom Image has blocked pool operation.
+        /// </summary>
         public const string OperationsOnImageRestrictedByPolicy = "OperationsOnImageRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on Network Security Group has blocked pool operation.
+        /// </summary>
         public const string OperationsOnNetworkSecurityGroupRestrictedByPolicy = "OperationsOnNetworkSecurityGroupRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on Load Balancer has blocked pool operation.
+        /// </summary>
         public const string OperationsOnLoadBalancerRestrictedByPolicy = "OperationsOnLoadBalancerRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on Virtual Network has blocked pool operation.
+        /// </summary>
         public const string OperationsOnVirtualNetworkRestrictedByPolicy = "OperationsOnVirtualNetworkRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on Private Link has blocked pool operation.
+        /// </summary>
         public const string OperationsOnPrivateLinkRestrictedByPolicy = "OperationsOnPrivateLinkRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Assigned policy on Private Endpoint has blocked pool operation.
+        /// </summary>
         public const string OperationsOnPrivateEndpointRestrictedByPolicy = "OperationsOnPrivateEndpointRestrictedByPolicy";
-
+        
+        /// <summary>
+        /// Requested VM size is not compatible.
+        /// </summary>
         public const string IncompatibleVMSize = "IncompatibleVMSize";
-
-        public const string IncomptabileForAcceleratedNetworking = "IncomptabileForAcceleratedNetworking";
-
+        
+        /// <summary>
+        /// Desired number of nodes could not be allocated due to a stop resize operation
+        /// </summary>
         public const string UnsupportedVMSize = "UnsupportedVMSize";
-
+        
+        /// <summary>
+        /// Requested VM size is not supported.
```

Comment:
Is this generally quota or feature flag restricted?

Reply:
This is when the account is missing the feature flag